drivers/sdhc/brcm_bcm2712_sdhci.c

```diff
@@ -453,7 +461,18 @@ static int bcm2712_sdhci_init(const struct device *dev)
 	return 0;
 }
 
+#if defined(CONFIG_BRCM_BCM2712_XFER_ADMA)
+#define BRCM_BCM2712_ADMA_DESCS(inst)                                                              \
+	static struct sdhc_adma_desc                                                               \
+		adma_descs_##inst[CONFIG_BRCM_BCM2712_ADMA_DESCS_NUM] __nocache;
```

Comment:
I would prefer something like this:
```
#ifdef CONFIG_NOCACHE_MEMORY
#define __NOCACHE __attribute__((__section__(".nocache")))
#else
#define __NOCACHE
#endif

static struct sdhc_adma_desc adma_descs_##inst[CONFIG_BRCM_BCM2712_ADMA_DESCS_NUM] __NOCACHE;
```

Reply:
It's already done in this way in Zephyr core, see `section_tags.h`:
```
#if defined(CONFIG_NOCACHE_MEMORY)
#define __nocache __in_section_unique(_NOCACHE_SECTION_NAME)
#define __nocache_noinit __nocache
#else
#define __nocache
#define __nocache_noinit __noinit
#endif /* CONFIG_NOCACHE_MEMORY */
```